esp32-mock-generator/src/main.cpp

```
@@ -0,0 +1,319 @@
+// MQTT Connection
+// ============================================================================
+void connectMQTT() {
+  int attempts = 0;
+  while (!mqttClient.connected() && attempts < 5) {
+    Serial.printf("→ Connecting to MQTT broker %s:%d ", MQTT_BROKER_IP, MQTT_PORT);
+
+    if (mqttClient.connect(MQTT_CLIENT_ID)) {
+      Serial.println("✓");
+      Serial.printf("  Publishing to: %s\n", MQTT_TOPIC);
+      return;
+    } else {
```

Clang (14.0.6)

[warning] do not use 'else' after 'return'

(readability-else-after-return)